### ISSUE TITLE
Schema Sync requires MySQL version 5.0+ (source is v10.1.0-MariaDB)

### ISSUE THREAD

**Issue by arteymix** [mmatuson/SchemaSync#32]
It works well on MariaDB by the way ;)

**Comment by jon-flashtalking** [comment]
Are you going to fix this ?

**Comment by mmatuson** [comment]
No. I'm not actively maintaining this project anymore, but am open to PR's to fix any issues.

**Comment by stormweaver** [comment]
Actually, if you look at the source code, you just have to comment out the lines that are throwing the exceptions, then it works as a charm. Mariadb has the same API as mysql, so this is the only problem. Just my 2 cents. :)
